form_designer/contrib/cms_plugins/form_designer_form/migrations/0001_initial.py

```diff
@@ -2,10 +2,19 @@
 from __future__ import unicode_literals
 
 from django.db import migrations, models
+from cms import __version__
```

Comment:
It would be nicer to import just `import cms` and use it with `cms.__version__` to avoid having misleading variable name (bare `__version__`) declared in this migration file's scope.